public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -134,6 +134,14 @@ const defaults: any = {
   savedQueryLink: null,
 };
 
+const autoMigrateAngular: Record<string, string> = {
+  graph: 'timeseries',
```

Comment:
maybe? not sure if change is necessary 
```suggestion
  'graph': 'timeseries',
```

Reply:
🤣 seems fine, but will immediately get changed back by prettier